### ISSUE TITLE
Export button text not visible with ‘differentiate without color’ accessibility option

### ISSUE THREAD

**Issue by jbmorley** [inseven/symbolic#256]
https://mastodon.gamedev.place/@gingerbeardman/113271288030777622

<img width="111" alt="e0ecbbc2b2deb714" src="https://github.com/user-attachments/assets/bfe1b682-f913-41dd-8a5c-aa3a119af025">

**Comment by gingerbeardman** [comment]
Curious:

<img width="827" alt="Screen shot 2024-10-08 at 22 58 30" src="https://github.com/user-attachments/assets/8a1d607b-8fc7-43d0-8dab-35e31f242d9e">

**Comment by gingerbeardman** [comment]
Also button is fine in Dark mode

<img width="138" alt="image" src="https://github.com/user-attachments/assets/af2948db-12c4-459e-8d5b-ca3c380129fc">

**Comment by jbmorley** [comment]
That's completely fascinating. I'm explicitly making the text white in both scenarios. I wonder if it's a SwiftUI bug. There's no shortage of them. I'll dig a bit.

**Comment by jbmorley** [comment]
On further investigation, the text appears to disappear when toggling the differentiate without color accessibility option, but it's fine again after relaunching.

**Comment by gingerbeardman** [comment]
Confirming as fixed ✨
